source/System.cpp

```diff
@@ -209,6 +211,12 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 			child.PrintTrace("Error: Expected key to have a value:");
 			continue;
 		}
+		else if(remove && child.Size() == 2 && key == "linger time")
+			lingerTime = -1;
```

Comment:
isnt this already taken care of right above?

Reply:
Maybe? I'll try removing it and see what happens.

Reply:
It is gone now.